lib/driver.js

```diff
@@ -13,7 +13,6 @@ const DEVICE_PORT = 4724;
 class AndroidDriver extends BaseDriver {
   constructor (opts = {}, shouldValidateCaps = true) {
     super(opts, shouldValidateCaps);
-
     this.desiredConstraints = desiredConstraints;
```

Comment:
I think this should be `this.desiredCapConstraints = desiredContraints;`

https://github.com/appium/appium-base-driver/blob/master/lib/driver.js#L45